src/server/rollbar.js

```diff
@@ -30,6 +30,15 @@ function Rollbar(options, client) {
   }
 }
 
+Rollbar.instance = null;
```

Comment:
Can we add static methods to the Rollbar Object and proxy those calls to the singleton?

E.g.
```js
Rollbar.debug(...) {
    if (Rollbar.instance) {
        return Rollbar.instance.debug(...)
    } else {
        throw Exception('Rollbar not initialized');
    }
}
```